llvm/lib/Target/AIE/AIE2MultiSlotPseudoInstrInfo.td

```diff
@@ -31,23 +37,68 @@ let isMoveImm = 1, isReMaterializable = 1, isAsCheapAsAMove = 1, Itinerary = II_
     hasSideEffects = false, mayLoad = false, mayStore = false in {
 
   // To move 10bit imm to R/lc type reg.
-  def MOV_RLC_imm10_pseudo : MultiSlot_Pseudo< (outs eRLC:$dst) , (ins simm10:$i),
-      "mov_rlc_imm10_pseudo ", "$dst, $i", [MOVA_lda_cg, MOVX_alu_cg, MOV_mv_cg, MOVXM_lng_cg]>;
+  def MOV_RLC_imm10_pseudo : MultiSlot_Pseudo< (outs eRLC:$dst), (ins simm10:$i),
+      "mov_rlc_imm10_pseudo ", "$dst, $i",
+      [MOVA_lda_cg, MOVX_alu_cg, MOV_mv_cg, MOVXM_lng_cg]>;
```

Comment:
Nit: Please avoid committing unnecessary diff lines.

Reply:
Sure, was suggested to have lines within  80 char. So fixed the whole file.